apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PairAssertionSamples.kt

```diff
@@ -0,0 +1,52 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class PairAssertionSamples {
+
+    @Test
+    fun first() {
+        val pair = 1 to "one"
+
+        expect(pair).first.isLessThan(expected = 2)
+
+        fails {
+            expect(pair).first.isGreaterThan(expected = 2)
+        }
+    }
+
+    @Test
+    fun firstAsFunction() {
+        val pair = 1 to "one"
+
+        expect(pair).first { isLessThan(expected = 2) }
+
+        fails {
+            expect(pair).first { isGreaterThan(expected = 2) }
+        }
+    }
+
+    @Test
+    fun second() {
+        val pair = 1 to "one"
+
+        expect(pair).second.isSameAs(expected = "one")
```

Comment:
Adapt to the review comment of first:
- use secondFeature for the overload which does not expect an assertionCreater-lambda
- never use `expected = ...` must have been a misunderstanding

Reply:
ok